core/api/src/main/java/com/alipay/sofa/rpc/message/AbstractResponseFuture.java

```diff
@@ -316,4 +316,4 @@ protected void setDoneTime() {
     public long getElapsedTime() {
         return doneTime - genTime;
     }
-}
+}
```

Comment:
Same problem.

Reply:
Do you know why unit testing is unstable? Sometimes through unit testing, occasionally with exceptions

Reply:
@EvenLjj  We need to make our unit test stable. It is disgusting to try it again and again :( . How about add a task to solve it ?

Reply:
OK, some single tests require reconstruction, we will kick of.